### ISSUE TITLE
Random seed in tf.keras.initializers.RandomNormal does not work in Tensorflow 2.7.0

### ISSUE THREAD

**Issue by lubin-liu** [tensorflow/tensorflow#53302]
<em>Please make sure that this is a bug. As per our
[GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md),
we only address code/doc bugs, performance issues, feature requests and
build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): Yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Google Colab
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: N/A
- TensorFlow installed from (source or binary): 
- TensorFlow version (use command below): 2.7.0
- Python version: 3.7.12
- Bazel version (if compiling from source): N/A
- GCC/Compiler version (if compiling from source): N/A
- CUDA/cuDNN version: N/A
- GPU model and memory: N/A

You can collect some of this information using our environment capture
[script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
You can also obtain the TensorFlow version with:
1. TF 1.0: `python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"`
2. TF 2.0: `python -c "import tensorflow as tf; print(tf.version.GIT_VERSION, tf.version.VERSION)"`

**Describe the current behavior**
Setting the seed in tf.keras.initializers.RandomNormal(seed = [some integer]) does not guarantee that the same sequence is generated when constructed with the same seed value.

**Describe the expected behavior**
Setting the seed in tf.keras.initializers.RandomNormal(seed = [some integer]) should guarantee that the same sequence is generated when two initializers are constructed with the same seed value.

**[Contributing](https://www.tensorflow.org/community/contribute)**

- Do you want to contribute a PR? (yes/no): no
- Briefly describe your candidate solution(if contributing): n/a

**Standalone code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate
the problem. If possible, please share a link to Colab/Jupyter/any notebook.

Using tensorflow version 2.7.0:
https://colab.research.google.com/drive/1paNe8kBRzwkrZHBi7QF0yhEX8pXO0f1w?usp=sharing

However, using tensorflow 2.6.0, this issue does not occur (note one has to restart the notebook after the pip install in the first cell before running the second cell):
https://colab.research.google.com/drive/10KrXpJtxiaAmnI2vKa61hDNheqjiJyu9?usp=sharing

**Other info / logs** Include any logs or source code that would be helpful to
diagnose the problem. If including tracebacks, please include the full
traceback. Large logs and files should be attached.

**Comment by mohantym** [comment]
Hi @lubin-liu! As this [document ](https://keras.io/api/layers/initializers/)suggests ,They are meant to give random weight values to input layer. So This is working as expected. Attaching [Gist ](https://colab.research.google.com/gist/mohantym/84541e4f6348446445e395410304825c/tf-keras-initializer-random-seed-test-using-version-2-7-0.ipynb#scrollTo=0UOgrynLV7Vh)for reference.

**Comment by mohantym** [comment]
Ok @lubin-liu ! Thanks for reporting this bug . 
Please post this issue on [keras-team/keras repo](https://github.com/keras-team/keras/issues) too.
To know more see;
[https://discuss.tensorflow.org/t/keras-project-moved-to-new-repository-in-https-github-com-keras-team-keras/1999](https://discuss.tensorflow.org/t/keras-project-moved-to-new-repository-in-https-github-com-keras-team-keras/1999) . Thanks!

**Comment by lubin-liu** [comment]
Thanks! I have done so here:

https://github.com/keras-team/keras/issues/15755

**Comment by mohantym** [comment]
Ok @lubin-liu ! Please close this issue here as it will be tracked in [#15755](https://github.com/keras-team/keras/issues/15755). Thanks!
